src/features/offer/components/OfferTags/OfferTags.native.test.tsx

```diff
@@ -13,4 +14,13 @@ describe('OfferTags component', () => {
     expect(screen.getByText('Tag2')).toBeOnTheScreen()
     expect(screen.getByText('Tag3')).toBeOnTheScreen()
   })
+
+  it('should display tags within the specified number of lines', () => {
+    const tagsLines = 2
+    render(<OfferTags tags={tags} tagsLines={tagsLines} />)
+
+    expect(screen.getByTestId('tagsContainer')).toHaveStyle({
+      maxHeight: (TAG_HEIGHT + MARGIN_BOTTOM_TAGS) * tagsLines,
```

Comment:
la vérification me parait trop proche de l'implem, je préfèrerais voir la valeur réelle

```suggestion
      maxHeight: 42,
```

ça fait probablement pas 42, j'ai pas compté, mais tu as l'idée